doc/source/Resources/PyAEDTInstallerFromDesktop.py

```diff
@@ -42,17 +49,21 @@ def run_pyinstaller_from_c_python(oDesktop):
         return_code = run_command(" ".join(command))
         err_msg = "There was an error while installing PyAEDT. Refer to the Terminal window where AEDT was launched " \
                   "from."
-    if is_windows:
-        venv_dir = os.path.join(os.environ["APPDATA"], "pyaedt_env_ide", "v" + version)
-        python_exe = os.path.join(venv_dir, "Scripts", "python.exe")
-    else:
-        venv_dir = os.path.join(os.environ["HOME"], "pyaedt_env_ide", "v" + version)
-        python_exe = os.path.join(venv_dir, "bin", "python")
+
     if str(return_code) != "0":
         oDesktop.AddMessage("", "", 2, err_msg)
         return
     else:
         oDesktop.AddMessage("", "", 0, "PyAEDT virtual environment created.")
+
+    # Add PyAEDT tabs in AEDT
+    # Virtual environment path and Python executable
+    if is_windows:
+        venv_dir = os.path.join(os.environ["APPDATA"], "pyaedt_env_ide", python_version.replace(".", "_"))
```

Comment:
You might want to define VENV_DIR_PREFIX at the begining of the file (`os.path.join(os.environ["APPDATA"], "pyaedt_env_ide"` OR the other name i propose in the following comment) as it used in multiple places of the file.
This would enforce the idea of my following comment for having an agnostic venv_dir destination.